gordo_components/builder/build_model.py

```diff
@@ -45,10 +46,13 @@ def build_model(
         Mapping of the Dataset to initialize, following the same logic as model_config.
     metadata: dict
         Mapping of arbitrary metadata data.
+    cv_mode: 
+        String which enables three different modes: Only printing cv_scores after cross validation, build the model i.e.
+        no cross validation or do a full build. Default is full build with cross validation.
 
     Returns
     -------
-        Tuple[sklearn.base.BaseEstimator, dict]
+        Tuple[Optional[sklearn.base.BaseEstimator], dict]
```

Comment:
I know it wasn't updated before, but just made me notice that we didn't type this return in the actual function.
`-> Tuple[...blah blah]`